tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Controllers/OrderItemsControllerTests.cs

```diff
@@ -47,9 +47,11 @@ public static void Constructors_VerifyGuardClauses()
         [Test]
         [InMemoryDbAutoData]
         public static async Task ListAsync_OrderDoesNotExist_ReturnsNotFound(
+            [Frozen] Mock<IOrderItemService> service,
             CallOffId callOffId,
             OrderItemsController controller)
         {
+            service.Setup(o => o.GetOrderItems(callOffId, It.IsAny<CatalogueItemType?>())).ReturnsAsync((List<OrderItem>)null);
```

Comment:
Should this not be tested for other `CatalogueItemTypes` besides `null` also? 
Suggest using a random value from possible values if agree.

Reply:
With PR [424](https://github.com/nhs-digital-gp-it-futures/BuyingCatalogueOrdering/pull/424/files)
Added one more test case with Non-nullable CatalogueItemTypes